tests/parser-test-files/bad-files/define-class.4.error

```diff
@@ -1,7 +1,7 @@
 error: Malformed method definition
-  --> test:5:7
+  --> test:5:4
    |
  4 |  (define-class (C :a)
    |                ------ in this class definition
  5 |   (m :a :b))
-   |         ^^ unexpected trailing form
+   |      ^^ expected docstring
```

Comment:
I know I wrote this, but I'm wondering whether this implies that a docstring is necessary, as opposed to just the second element needing to be a docstring if there are three in the form.

Maybe `"unexpected trailing form"` on the third form is the proper error.

Reply:
Long story short, I wonder whether the first error is still more accurate and advantageous.

Reply:
That does correspond better to what seems to be going on with the program input, I'll change it.

Reply:
done!